AUTHORING_GUIDE.md

```diff
@@ -269,6 +316,25 @@ If a sample has testing requirements that differ from its runtime requirements
 testing libraries), the testing requirements may be listed in a separate
 `requirements-test.txt` file instead of the main `requirements.txt` file.
 
+#### Developing samples for un-released changes
+
+Pip has VCS support. Use the branch name or commit hash instead of the package name.
+
+https://pip.pypa.io/en/stable/cli/pip_install/#vcs-support
```

Comment:
```suggestion
Pip has [VCS support](https://pip.pypa.io/en/stable/cli/pip_install/#vcs-support). Use the branch name or commit hash instead of the package name.

```